resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/config/ApplicationConfiguration.java

```diff
@@ -182,7 +181,7 @@ public StreamingResourceServiceProxy resourceServiceProxy(
     @ConditionalOnProperty(prefix = "resource", name = "implementation", havingValue = "simple")
     @Qualifier("impl")
     public ResourceService simpleResourceService() {
-        return new SimpleResourceService(dataServiceName, defaultResourceType);
+        return new SimpleResourceService(dataServiceName);
     }
```

Comment:
Bad merge, this is needed

Reply:
There three differences between this branch and the develop branch.  All were related to the defaultResourceType.   The have been added to the class.  The consists of: 

@Value("${resource.defaultType}")
    private String defaultResourceType;

and 
@Value("${resource.defaultType}")
    private String defaultResourceType;

        return new SimpleResourceService(dataServiceName, defaultResourceType);